0-ad-history-mod-package/simulation/templates/structures/ptol/farmstead.xml

```diff
@@ -6,6 +6,9 @@
   <Identity>
     <Civ>ptol</Civ>
     <SpecificName>Epoikion</SpecificName>
+    <History>
+        A successful farm year in Egypt was dependent on the Nile river. Unfortunately, some years the river couldn't provide the necessary water, causing a drought. In plentiful years Egyptians stored excess produce in case the next year they did not have the same fortune.
```

Comment:
```suggestion
        A successful farm year in Egypt was dependent on the Nile River. Unfortunately, some years the river couldn't provide the necessary water, causing a drought. In plentiful years Egyptians stored excess produce in case the next year they did not have the same fortune.
```